lib/workers/repository/update/pr/changelog/release-notes.ts

```diff
@@ -97,40 +97,65 @@ export async function getReleaseNotes(
   project: ChangeLogProject,
   version: string
 ): Promise<ChangeLogNotes | null> {
-  const { baseUrl, depName, repository } = project;
+  const { depName, repository } = project;
   logger.trace(`getReleaseNotes(${repository}, ${version}, ${depName})`);
   const releaseList = await getCachedReleaseList(project);
   logger.trace({ releaseList }, 'Release list from getReleaseList');
   let releaseNotes: ChangeLogNotes | null = null;
-  for (const release of releaseList) {
-    if (
-      release.tag === version ||
-      release.tag === `v${version}` ||
-      release.tag === `${depName}-${version}` ||
-      release.tag === `${depName}_v${version}` ||
-      release.tag === `${depName}@${version}`
-    ) {
-      releaseNotes = release;
-      releaseNotes.url = baseUrl.includes('gitlab')
-        ? `${baseUrl}${repository}/tags/${release.tag}`
-        : `${baseUrl}${repository}/releases/${release.tag}`;
-      releaseNotes.body = massageBody(releaseNotes.body, baseUrl);
-      if (releaseNotes.body.length) {
-        try {
-          if (baseUrl !== 'https://gitlab.com/') {
-            releaseNotes.body = await linkify(releaseNotes.body, {
-              repository: `${baseUrl}${repository}`,
-            });
-          }
-        } catch (err) /* istanbul ignore next */ {
-          logger.warn({ err, baseUrl, repository }, 'Error linkifying');
-        }
-      } else {
-        releaseNotes = null;
+  let matchedRelease: ChangeLogNotes | undefined | null = null;
```

Comment:
```suggestion
  let matchedRelease: ChangeLogNotes | undefined;
```

locks cleaner to me 🙃